admin_manual/configuration/user_auth_ldap_cleanup.rst

```diff
@@ -0,0 +1,69 @@
+=================
+LDAP User Cleanup
+=================
+
+LDAP User Cleanup is a new feature in the ``LDAP user and group backend`` 
+application. LDAP User Cleanup is a background process that automatically 
+searches the ownCloud LDAP mappings table, and verifies if the LDAP users are 
+still available. Any users that are not available are marked as ``deleted`` in 
+the ``oc_preferences`` database table. Then you can run a command to display 
+this table, displaying only the users marked as ``deleted``, and then you have 
+the option of removing their data from your ownCloud data directory.
+
+These items are removed upon cleanup:
+
+* Local ownCloud group assignments
+* User preferences (DB table ``oc_preferences``)
+* User's ownCloud home folder
+* User's corresponding entry in ``oc_storages``
+
+There are two prequisites for LDAP User Cleanup to operate:
+
+1. Set ``ldapUserCleanupInterval`` in ``config.php`` to your desired check 
+   interval in minutes. The default is 51 minutes.
+
+2. All configured LDAP connections are enabled and operating correctly. As users 
+   can exist on multiple LDAP servers, you want to be sure that all of your 
+   LDAP servers are available so that a user on a temporarily disconnected LDAP 
+   server is not marked as ``deleted``.
+   
+The background process examines 50 users at a time, and runs at the interval you 
```

Comment:
One question left might be: were are the users come from that are being examined? From the LDAP mappings table. Not sure this needs to be mentioned, as a informed user might guess it.